art/metrics/privacy/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Module providing metrics and verifications.
+"""
+from art.metrics.privacy.membership_leakage_metrics import PDTP
```

Comment:
How about if we change the module name from `membership_leakage_metrics` to `membership_leakage`? I`art.metrics.privacy.membership_leakage` would be shorter and only contain `metrics` once.